gradio/components.py

```diff
@@ -131,12 +124,25 @@ def restore_flagged_file(
         return {"name": file, "data": data}
 
     @classmethod
-    def get_all_shortcut_implementations(cls):
-        shortcuts = {}
+    def get_component_shortcut(cls, str_shortcut: str) -> (bool, Optional[Component]):
```

Comment:
It's redundant for this function to return two values. It should just return a component or `None`